app/web/template/layout/date.gohtml

```diff
@@ -2,17 +2,27 @@
   <div class="govuk-form-group {{ if index .top.Errors .name }}govuk-form-group--error{{ end }} govuk-!-margin-bottom-6">
     <fieldset class="govuk-fieldset" role="group" aria-describedby="{{ .name }}-hint">
       <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">{{ tr .top.App .label }}</legend>
+      {{ if .context }}
+        <p class="govuk-body">
+          {{ tr .top.App .context }}
+        </p>
+      {{ end }}
       {{ if .hint }}
         <div id="{{ .name }}-hint" class="govuk-hint">
           {{ tr .top.App .hint }}
         </div>
       {{ end }}
+
       {{ template "error-message" (errorMessage .top .name) }}
+      {{ template "error-message" (errorMessage .top (printf "%v-day" .name)) }}
+      {{ template "error-message" (errorMessage .top (printf "%v-month" .name)) }}
+      {{ template "error-message" (errorMessage .top (printf "%v-year" .name)) }}
```

Comment:
hmm, it doesn't explicitly state on the design system but I think we should only ever show one message per field, could we combine the messages?

Reply:
Ah cool missed this - good to know.